themes-default/slim/static/css/style.css

```diff
@@ -1437,7 +1441,7 @@ span.snatched b {
     font-weight: 800;
 }
 
-.displayShowTable {
+.displayShowTable > table {
```

Comment:
Yeah I need to. The tablesorter doesn't work nice with vue. And overwrites classes.

Reply:
Yup. We knew this was going to be an issue.
And I didn't see anything as good as tablesorter for Vue.

Reply:
We can have tablesorter with vue. I'm pretty confident

Reply:
And if not we can implemt it in native vue

Reply:
Don't know if this is a good option, but https://github.com/xaksis/vue-good-table

Reply:
That's basically a component that you pass an array with flat objects. And renders a complete table. I think thats why @sharkykh couldn't find anything as "powerfull" as table sorter.

Reply:
I think i'm going for vuetify data-tables. It's time for some change anyway. And this will give us the opportunity to add cards for mobile.

Reply:
@sharkykh I can't get it to recognize the v-data-table component. Is it maybe because of the way we've hacked in vue?

Reply:
> @sharkykh I can't get it to recognize the v-data-table component. Is it maybe because of the way we've hacked in vue?

Try adding it globally, like `Snotify`, for now:
https://github.com/pymedusa/Medusa/blob/597af4fb4159aecb9f6ba6973cb57568440d1ad6/themes-default/slim/views/layouts/main.mako#L174
https://github.com/pymedusa/Medusa/blob/597af4fb4159aecb9f6ba6973cb57568440d1ad6/themes-default/slim/src/index.js#L65
https://github.com/pymedusa/Medusa/blob/597af4fb4159aecb9f6ba6973cb57568440d1ad6/themes-default/slim/src/app.js#L33

Reply:
I tried this. Didn't work. Doesn't recognize the v-data-table. I can commit it so you can see for yourself.

Reply:
If I can't get vuetify to work I'll try https://github.com/matfish2/vue-tables-2/blob/master/README.md#nested-data